modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -186,6 +186,348 @@ Test_ProcessTimes(void)
 #undef SPIN_TIME
 }
 
+static
+void
+Test_ProcessBasicInformation(void)
+{
+    NTSTATUS Status;
+    ULONG Length;
+    PROCESS_BASIC_INFORMATION BasicInfo1;
+    PROCESS_BASIC_INFORMATION BasicInfo2;
+
+    /* Everything is NULL */
+    Status = NtQueryInformationProcess(NULL,
+                                       ProcessBasicInformation,
+                                       NULL,
+                                       0,
+                                       NULL);
+    ok_hex(Status, STATUS_INFO_LENGTH_MISMATCH);
+
+    /* Right size, invalid process handle */
+    Status = NtQueryInformationProcess(NULL,
+                                       ProcessBasicInformation,
+                                       NULL,
+                                       sizeof(PROCESS_BASIC_INFORMATION),
```

Comment:
Do you prefer keeping the explicit type in the sizeof, or would it be possible to use `sizeof(BasicInfo1)` for example?

Reply:
Initially, I keep it consistent with `ProcessTimes`, but since `ProcessWx86Information` also use sizeof(var) I think it's OK here.